src/approximations/importance.jl

```diff
@@ -83,3 +87,20 @@ function approximate_meancov(approximation::ImportanceSamplingApproximation, g::
 
     return m, v
 end
+
+is_point_mass_form_constraint(::ImportanceSamplingApproximation) = false
+default_form_check_strategy(::ImportanceSamplingApproximation)   = FormConstraintCheckEach()
+default_prod_constraint(::ImportanceSamplingApproximation)       = ProdGeneric()
+
+function constrain_form(approximation::ImportanceSamplingApproximation, distribution::DistProduct)
+    m, v = approximate_meancov(approximation, x -> pdf(getright(distribution), x), getleft(distribution))
+
+    a = m^2 / v
+    b = m / v
+    return GammaShapeRate(a, b)
+end
+
+constrain_form(::ImportanceSamplingApproximation, distribution::Any) = distribution
+
+make_form_constraint(::Type{ImportanceSamplingApproximation}, args...; kwargs...) =
```

Comment:
This should be removed in favor of separate `MomentsApproximationFormConstraint `